modules/swagger-codegen/src/main/resources/rust/model.mustache

```diff
@@ -0,0 +1,57 @@
+{{>partial_header}}
+{{#models}}
+{{#model}}
+{{#description}}
+// {{{classname}}} : {{{description}}}
```

Comment:
Should be `///` here and everywhere the api docs are expected.

Is `description` guaranteed to be a single line?